app/src/androidTest/java/org/mozilla/fenix/components/FxaServer.kt

```diff
@@ -0,0 +1,17 @@
+package org.mozilla.fenix.components
+
+import android.content.Context
+import mozilla.components.service.fxa.ServerConfig
+
+object FxaServer {
+    const val CLIENT_ID = "a2270f727f45f648"
```

Comment:
will this CLIENT_ID also be replaced by a dynamic one?  (same as:  https://github.com/mozilla-mobile/fenix/pull/5633/files/b6603729b13964d8f52aa06342a74601f2bf6cd3#r329536741)?  It's OK to hard code these if necessary, but it would be more brittle.  Esp. on the stage server

Reply:
This implementation is similar, using same CLIENT_ID for the non test part, the regular FxaServer.kt. But I can check if this can be changed